wordpress-mobile/WordPress-Android
# Add shared prefs for blogging reminders shown

Fixes #14711 

This PR adds an option to app prefs that stores the fact that the blogging reminders prompt was already shown to the user. I've extracted the flag and the app prefs option into a `manager` class and added basic tests for it and the view model.

To test:
- Make sure the BloggingReminders flag is turned off
- Create and publish a new post
- Notice there is no bottom sheet visible
- Turn the Blogging reminders flag on and restart the app
- Create and publish a new post
- Notice the bottom sheet
- Dismiss it
- Create and publish a new post
- Notice the bottom sheet is not shown again

## Regression Notes
1. Potential unintended areas of impact
- None

PR submission checklist:
